packages/patternfly-4/react-charts/src/components/ChartBar/examples/ChartBar.md

```diff
@@ -88,52 +93,54 @@ import { Chart, ChartBar, ChartGroup, ChartThemeColor, ChartVoronoiContainer } f
       </ChartGroup>
     </Chart>
   </div>
-</div>
+)
 ```
 
-```js title=Multi-color-(ordered),-horizontal-bar-chart-with-bottom-left-aligned-legend
+```js title=Multi--color-with-bottom--left--aligned-legend
```

Comment:
The title was "Multi-color (ordered), horizontal bar chart with bottom-left aligned legend" -- the 'ordered' / 'unordered' descriptions are an important distinction here for mutil-color charts.

Also, "bottom--left--aligned" should be "bottom--left-aligned"